src/components/WhatsNew.vue

```diff
@@ -26,57 +26,56 @@
                 Water Data Blog </span>
             </h2></a>
         </div>
-        
         <div id="viz-text">
-         <!-- blog list item -->
           <div class="text-container">
+            <!-- blog list item -->
             <li>
               <span class="date-text">
-                01/04/2023:
+                01/11/2023:
               </span> 
-              A New Take on the Water Cycle
+              Origin and development of a Snowflake Map
               <a
-                href="https://waterdata.usgs.gov/blog/water-cycle-release/"
+                href="https://waterdata.usgs.gov/blog/snow-tiles-demo/"
                 target="_blank"
               >
                 <span class="arrow">
                   Read &#8594;
                 </span>
               </a>
+              <!-- include thumbnail of most recent blog post -->
+              <div class="thumbnail-container">
+                <img 
+                  class="blog-thumbnail"
+                  src="https://labs.waterdata.usgs.gov/visualizations/thumbnails/SnowTiles_thumbnail.png" 
+                  alt="A network with water droplets for nodes. The biggest node says 'USGS Data Science' and stemming from it are nodes for data viz, analytics, machine learning, and reproducible data assembly, the four specializations of the Data Science Branch"
+                >
+              </div>
```

Comment:
this is consistent with what I've been doing, but do you think it's too repetitive to have the same image twice on landing? 

<img width="1459" alt="image" src="https://user-images.githubusercontent.com/17803537/213590725-b4a22538-918b-4e97-b0a4-edaf7aa7100e.png">

Reply:
good point - I'll switch it to the water cycle blog for now